livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java

```diff
@@ -75,13 +75,13 @@ public Integer createSession(InterpreterContext context, String kind) throws Exc
 
       String confData = gson.toJson(conf);
 
-      String json = executeHTTP(property.getProperty("zeppelin.livy.url") + "/sessions",
+      String json = executeHTTP(property.getProperty("livy.host.url") + "/sessions",
```

Comment:
my thought would be this:
- any property needed by Zeppelin (such as to communicate with another server), it should start with "zeppelin.*"
- any property not used by Zeppelin but by a separate server or framework (eg. spark), it should start with "spark.*"
- the reason is the interpreter can then pass-through properties as-is, for example, spark interpreter can pass all "spark.*" property to SparkConf (and skip all the zeppelin.* ones!)

I think we try to roughly follow that pattern (at least as much as I have seen)